BluetoothLeChat/app/src/main/res/layout/fragment_device_scan.xml

```diff
@@ -24,37 +23,60 @@
         android:id="@+id/scanning"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
         android:layout_gravity="center"
+        android:orientation="vertical"
         android:visibility="gone">
 
         <ProgressBar
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_gravity="center"/>
+            android:layout_gravity="center" />
 
         <TextView
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
+            android:paddingTop="16dp"
```

Comment:
should these all be resources?

Reply:
yes. The project itself, which I inherited, has a number of those. I plan a follow on PR to fix them all.